app/models/watchlist_instance.rb

```diff
@@ -345,6 +334,7 @@ def delete_watchlist_instance
       raise "Unable to delete a watchlist instance that is not pending or archived #{id}"
     end
   end
+  # rubocop:enable Style/GuardClause
```

Comment:
What are these two rubocop:* comment lines for?

Reply:
This two lines makes the linter ignore style/guardclause rule in between the two lines. 
I did this because the suggestion provided by the linter was not sensible to adopt.